packages/accounts-app/src/Validators/ValidatorRow.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { AccountId, Balance, Exposure, Option, StakingLedger } from '@polkadot/types';
+import { formatBalance } from '@polkadot/util';
+import { AppContext } from '@substrate/ui-common';
+import { AddressSummary, FadedText, Stacked, Table, Icon } from '@substrate/ui-components';
+import BN from 'bn.js';
+import H from 'history';
+import { fromNullable, some } from 'fp-ts/lib/Option';
+import React, { useContext, useEffect, useState } from 'react';
+import { combineLatest, Observable, Subscription } from 'rxjs';
+import { first, switchMap } from 'rxjs/operators';
+import { Loader } from 'semantic-ui-react';
+
+import { OfflineStatus } from '../Accounts/types';
+import { ConfirmNominationDialog } from './ConfirmNominationDialog';
+
+interface Props {
+  history: H.History;
+  offlineStatuses?: OfflineStatus[];
+  validator: AccountId;
+}
+
+export function ValidatorRow (props: Props) {
+  const { offlineStatuses, validator } = props;
+  const { api, keyring } = useContext(AppContext);
+  const [nominations, setNominations] = useState<[AccountId, Balance][]>([]);
+  const [nominees, setNominees] = useState<AccountId[]>();
+  const [offlineTotal, setOfflineTotal] = useState<BN>(new BN(0));
+  const [loading, setLoading] = useState<boolean>(false);
+
+  const currentAccount = location.pathname.split('/')[2];
+
+  useEffect(() => {
+    setLoading(true);
+    const subscription: Subscription = (
+        api.queryMulti([
```

Comment:
probably could moe this stuff to staking context too, but again that's for another PR